server/db_accessor.h

```diff
@@ -10,6 +10,9 @@
 #include <ozo/connection_info.h>
 #include <ozo/connection_pool.h>
 #include <ozo/connection.h>
+#include <ozo/shortcuts.h>
```

Comment:
Any reason why these are moved to the header file? I think it's generally encouraged to include in cpp if possible and have each file include what they need (except .h -> .cpp). Transitive includes are discouraged too. https://softwareengineering.stackexchange.com/questions/262019/is-it-good-practice-to-rely-on-headers-being-included-transitively

Reply:
I moved them to db_accessor.h because server_gamedata needs ozo connection header files. After this movement, I included ad_accessor.h in server_gamedata.h. Another way, as you suggested, is to include necessary files in server_gamedata.cpp.